romana-install/group_vars/all/stack

```diff
@@ -2,5 +2,4 @@ stack_password: "secrete"
 romana_ssh_key: "romana_id_rsa"
 compute_nodes: 1
 
-romana_cidr: 10.0.0.0/8
```

Comment:
It's duplicated in https://github.com/romana/romana/blob/master/romana-install/group_vars/all/romana#L11-L12